src/EditorFeatures/Core.Wpf/Adornments/AbstractAdornmentManager.cs

```diff
@@ -51,9 +51,9 @@ internal abstract class AdornmentManager<T> where T : GraphicsTag
         /// It happens when another region of the view becomes visible or there is a change in tags.
         /// For us the end result is the same - get tags from tagger and update visuals correspondingly.
         /// </summary>        
-        protected abstract void UpdateSpans_CallOnlyOnUIThread(NormalizedSnapshotSpanCollection changedSpanCollection, bool removeOldTags);
+        protected abstract void AddAdornmentsToAdornmentLayer_CallOnlyOnUIThread(NormalizedSnapshotSpanCollection changedSpanCollection);
```

Comment:
this is the part i think i'd prefer be in a different PR.  let's try to get all the core adornment manager code fixed.  and then this PR only adds the inline error manager, it doesn't update other managers.

Reply:
separated: https://github.com/dotnet/roslyn/pull/54700